README.md

```diff
@@ -154,15 +229,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+jonathan = Person.new(name: "Jonathan", age: 35)
```

Comment:
:+1: but remember that `.new` doesn't automatically save something to a database. You would need to follow that up with `.save` or use `.create` instead to do it all in one step.